### PR TITLE
VDM changes for remarketing lists in destination transform.js

### PR DESCRIPTION
**Pull request by Vikas26021999**
## VDM changes for remarketing lists

> Changes to use mapped fields instead of destination schema

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by RanjeetMishra — COMMENTED** [review]
@Vikas26021999 Lets add brief 2 lines about dev testing we did?

File: v0/destinations/google_adwords_remarketing_lists/transform.js [hunk 1]
```diff
@@ -20,6 +21,26 @@ const {
...
 
+const { MappedToDestinationKey } = require("../../../constants");
+
+const getSchemaForEventMappedToDest = message => {
+  const mappedSchema = get(message, "context.destinationFields");
+  if (!mappedSchema) {
+    throw new CustomError(
+      "context.destinationFields is required property for events mapped to destination ",
+      400
+    );
```
Comment (ItsSudip) [inline v0/destinations/google_adwords_remarketing_lists/transform.js:32]:
```suggestion
    throw new ErrorBuilder().setMessage(
      "context.destinationFields is required property for events mapped to destination ").setStatus(400).build();
```

**Comment by Vikas26021999** [comment]
> @Vikas26021999 Lets add brief 2 lines about dev testing we did?

image
Successful syncs after changing the transformer to `GA-Remarketing-Lists-VDM-Changes` on hosted-dev

**Review by ItsSudip — DISMISSED** [review]

File: v0/destinations/google_adwords_remarketing_lists/transform.js [hunk 2]
```diff
@@ -20,6 +21,24 @@ const {
   TYPEOFLIST
 } = require("./config");
 
+const { MappedToDestinationKey } = require("../../../constants");
+
+const getSchemaForEventMappedToDest = message => {
```
Comment (ashishRudder) [inline v0/destinations/google_adwords_remarketing_lists/transform.js:26]: we have the same piece of code for `fb_custom_audience`. Can it be carved out and moved to util and be used anywhere else.

Comment (ashishRudder) [reply v0/destinations/google_adwords_remarketing_lists/transform.js:26]: Let's mention the new method used for VDM in the notion task or a separate doc so we don't add any redundant code.

**Pull request closed without merging**